shell/platform/fuchsia/flutter/flatland_connection.cc

```diff
@@ -86,26 +86,26 @@ void FlatlandConnection::DoPresent() {
 
 // This method is called from the UI thread.
 void FlatlandConnection::AwaitVsync(FireCallbackCallback callback) {
+  const fml::TimePoint now = fml::TimePoint::Now();
```

Comment:
Could you add a comment explaining why you moved this here?

Since it's outside of the mutex, we could wait for some time inside of the mutex lock before executing the code below.  At that point the "now" variable wouldnt be "now" anymore - it would represent some time in the past.

Was that your intention here?